src/Core/src/Hosting/Dispatching/AppHostBuilderExtensions.cs

```diff
@@ -8,6 +13,8 @@ namespace Microsoft.Maui.Hosting
 {
 	public static partial class AppHostBuilderExtensions
 	{
+		const string androidEnvVarFilePath = "/data/local/tmp/ide-launchenv.txt";
```

Comment:
What puts this file on disk? We will need to be sure the code that reads this file is completely trimmed away in `Release` mode or use `@(AndroidEnvironment)` instead, as it should already be safe:

* https://github.com/dotnet/android/blob/1ff565829f5f50972971f8b87a3bd55eb3f2a6bb/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets#L1744-L1760

Reply:
Removed reference to this file completely as discussed offline